examples/device-dashboard/net.c

```diff
@@ -130,11 +130,13 @@ static void handle_login(struct mg_connection *c, struct user *u) {
 }
 
 static void handle_logout(struct mg_connection *c) {
-  mg_http_reply(c, 200,
-                "Set-Cookie: access_token=; Path=/; "
-                "Expires=Thu, 01 Jan 1970 00:00:00 UTC; "
-                "Secure; HttpOnly; Max-Age=0; \r\n",
-                "true\n");
+  char cookie[256];
+  mg_snprintf(cookie, sizeof(cookie),
+              "Set-Cookie: access_token=; Path=/; "
+              "Expires=Thu, 01 Jan 1970 00:00:00 UTC; "
+              "%sHttpOnly; Max-Age=0; \r\n",
+              c->is_tls ? "Secure; " : "");
+  mg_http_reply(c, 200, cookie, "true\n");
 }
```

Comment:
DO this in `handle_login()` too.

Reply:
`Secure` wasn't there in `handle_login()` before, only in `handle_logout()`; could this have been the issue ?

Reply:
It was not there because I've removed it at some stage, but forgot to remove in handle_logout